.github/workflows/release-manifests.yml

```diff
@@ -0,0 +1,35 @@
+name: Release Manifest
+
+# This workflow should be run after the integration is released into a new infrastructure-bundle image version
+# or just want to bump the bundle version for the same integration version.
+
+on:
+  workflow_dispatch:
+    inputs:
+      bundleVersion:
+        description: 'Version of Infra Bundle image without `v` prefix e.g. 2.6.4'
+        required: true
+      releaseVersion:
+        description: 'Version of nri-ecs without `v` prefix e.g. 1.3.1'
```

Comment:
thanks for specifying the "without v", I would have done that mistake for sure 😢

Reply:
+1, good UX choice there :)